docs/discord.rst

```diff
@@ -21,43 +21,43 @@ Creating a Bot account is a pretty straightforward process.
     .. image:: /images/discord_create_app_form.png
         :alt: The new application form filled in.
 
-5. Create a Bot User by navigating to the "Bot" tab and clicking "Add Bot".
+5. Create a Bot User Account by navigating to the "Bot" tab and clicking "Add Bot".
```

Comment:
```suggestion
5. Create a Bot User account by navigating to the "Bot" tab and clicking "Add Bot".
```

Reply:
It's kind of like saying just "User Account", so I thought adding the capital letter would be better.

Reply:
I understand it as an account for a Bot User, where account is just like a normal account and doesn't need to be capitalized.

Reply:
Ah, in that case then, I'll change it back.